src/h5web/providers/jupyter/api.ts

```diff
@@ -13,14 +13,13 @@ import type {
   JupyterMetaResponse,
 } from './models';
 import { nanoid } from 'nanoid';
-import { makeStrAttr, floatType } from '../mock/utils';
+import { makeStrAttr, floatType } from '../mock/data-utils';
```

Comment:
The mock data generation functions and mock types are moved to `data-utils` in order to have a file called `utils` for utilities like `findMockEntity` (previously `getEntityAtPath`. We have to have two separate files because of cyclic dependencies, but I think keeping the mock data generation utilities separate is best anyway.